### PR TITLE
feat(QTM-316): Adding typography mobile media query

### PR DESCRIPTION
**Pull request by matheusrpp**
## Description
https://jirasoftware.catho.com.br/browse/QTM-316

## Setup
to view the components behavior, run `yarn storybook`

## Review guide
- [ ] Coverage (coverage status should not regress)\
      - `yarn test:coverage`
- [ ] Unit tests (yarn test:components)
- [ ] Regression \
      - first start the storybook for regression tests(and keep it open): ` yarn test:regression:storybook`; \
      - then run the regression tests: `yarn test:regression`
- [ ] Code review

### Browsers review
- [ ] Layout review
  - [ ] Edge
  - [ ] Firefox
  - [ ] Chrome
  - [ ] Safari
  - [ ] Mobile
- [ ] Ux review validation

### CODE REVIEW

**Review by toncabral88 — APPROVED** [review]
## Review guide
- [x] Coverage (coverage status should not regress)\
      - `yarn test:coverage`
- [x] Unit tests (yarn test:components)
- [x] Regression \
      - first start the storybook for regression tests(and keep it open): ` yarn test:regression:storybook`; \
      - then run the regression tests: `yarn test:regression`
- [x] Code review

### Browsers review
- [x] Layout review

File: components/GlobalStyle/sub-components/Typography.csr.unit.test.jsx [hunk 1]
```diff
@@ -23,82 +23,48 @@ describe('Typography component', () => {
...
 
     expectCSSMatches(
-      `h2 {
-        font-size: 36px;
-        -webkit-letter-spacing: 0.5px;
-        -moz-letter-spacing: 0.5px;
-        -ms-letter-spacing: 0.5px;
-        letter-spacing: 0.5px;
-      }`,
+      `h2{font-size:32px;-webkit-letter-spacing:0.24px;-moz-letter-spacing:0.24px;-ms-letter-spacing:0.24px;letter-spacing:0.24px;}@media (min-width:1024px){h2{font-size:36px;-webkit-letter-spacing:0.54px;-moz-letter-spacing:0.54px;-ms-letter-spacing:0.54px;letter-spacing:0.54px;}}`,
```
Comment (rapahaeru) [inline components/GlobalStyle/sub-components/Typography.csr.unit.test.jsx:35]: if the font-size was wrong, updates here too.

**Review by rapahaeru — CHANGES_REQUESTED** [review]
## Review guide
- [x] Coverage (coverage status should not regress)\
      - `yarn test:coverage`
- [ ] Unit tests (yarn test:components)
- [x] Regression \
      - first start the storybook for regression tests(and keep it open): ` yarn test:regression:storybook`; \
      - then run the regression tests: `yarn test:regression`
- [ ] Code review

### Browsers review
- [ ] Layout review
- [ ] Ux review validation

File: components/GlobalStyle/sub-components/Typography.jsx [hunk 2]
```diff
@@ -24,33 +28,59 @@ h1, h2, h3, h4, h5, h6 {
...
 }
 
 h2 {
-  font-size: 36px;
-  letter-spacing: 0.5px;
+  font-size: 32px;
+  letter-spacing: 0.24px;
+
+  ${mediaQueries.medium`
+    font-size: 36px;
```
[the remark this answers is not in the document]
Comment (matheusrpp) [reply components/GlobalStyle/sub-components/Typography.jsx:45]: Mobile it's 32px and desktop it's 36px.
Desktop
Captura de tela de 2021-12-13 15-13-09
Mobile
Captura de tela de 2021-12-13 15-13-34

**Review by rapahaeru — APPROVED** [review]

**Pull request merged**